.github/workflows/publish_prerelease.yml

```
@@ -82,8 +82,10 @@ jobs:
         id: updated_version
         run: echo "version=$(yarn workspace @vkontakte/vkui --silent run --silent package:version)" >> $GITHUB_OUTPUT
 
-      - name: Creating doc for prerelease
+      - name: Build styleguide
         run: yarn run docs:styleguide:build --dist dist/${{ steps.updated_version.outputs.version }}
+      - name: Build storybook
+        run: yarn docs:storybook:build -o ../../styleguide/dist/${{ steps.updated_version.outputs.version }}/storybook
```

```suggestion
        run: yarn docs:storybook:build -o ../../styleguide/dist/${{ steps.updated_version.outputs.version }}/playground
```
получиться более абстрактно

---

А зачем абстрактно?

---

Не хотелось бы напрямую завязываться на название либы. "Завтра" появится новый инструмент, а в урле будет название старого. Да и `storybook` странное название

Я ещё по поводу `vkui-styleguide` тоже писал, что так себе название, но штош, для у него уже так сложилось